application/src/main/resources/application-default.yml

```diff
@@ -1,3 +1,22 @@
 hono:
   registration:
     filename: /home/hono/registration/device-identities.json
+  metric:
+    jvm:
+      memory: true
+      thread: true
+    reporter:
+      console:
+        active: false
+        period: 5000
+      graphite:
```

Comment:
So, if we agree, then please rebase and make the changes so that I can merge the PR ...